packages/glibc.rb

```diff
@@ -38,6 +39,10 @@ class Glibc < Package
     version Glibc_build237.version
     compatibility Glibc_build237.compatibility
     depends_on 'glibc_lib237'
+  when '2.41'
```

Comment:
Does this mean we need to update when the glibc version bumps to 2.42?  Would a `>=` comparison prevent requiring another update here?

Reply:
Logic changed.